scripts/verify-replace.sh

```diff
@@ -20,6 +20,7 @@ do
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
+    make pre-verify
```

Comment:
do we need to append the repo name to the error list also here as for the `verify-dependencies` ?

Reply:
agree with @mfrancisc - let's make sure that we add the repository into the list of the failed ones if the `pre-verify` fails

Reply:
if fails in the `pre-verify` we also need to skip the verify and go to the next repo I would say.

Reply:
updated it

Reply:
My understanding is if it fails in pre-verify it would automatically come out of that and go for another repo.. but i will test it confirm

Reply:
i have updated the logic to skip to next repo